frontend/src/containers/pages/AuthoringTool/CanvasSideBar/SceneSettings.jsx

```diff
@@ -70,6 +128,54 @@ export default function SceneSettings() {
               shrink: !!currentScene?.time,
             }}
           />
+          <FormControl fullWidth>
+            <CustomInputLabel>Scene Role(s)</CustomInputLabel>
+            <Select
+              className={styles.selectInput}
+              MenuProps={{
+                getContentAnchorEl: null,
+              }}
+              value={selectedRoles}
+              onChange={(event) => setSelectedRoles(event.target.value)}
+              renderValue={(selected) =>
+                selected.length === roleList.length
+                  ? "All"
+                  : selected.join(", ")
+              }
+            >
+              {roleList && roleList.length > 0 ? (
+                <div style={{ display: "flex", flexDirection: "column" }}>
+                  <FormControlLabel
+                    control={
+                      <CustomCheckBox
+                        defaultChecked
+                        checked={allChecked}
+                        onChange={handleAllToggle}
+                      />
+                    }
+                    label="All"
+                  />
+                  {roleList.map((role, index) => (
+                    <FormControlLabel
+                      control={
+                        <CustomCheckBox
+                          defaultChecked
+                          checked={checked[index]}
+                          onChange={() => handleCheckboxChange(index)}
+                        />
+                      }
+                      label={role}
+                    />
+                  ))}
+                </div>
+              ) : (
+                <Typography className={styles.menuItem}>
+                  This scenario currently does not accommodate roles. Please
+                  upload a CSV file first under Manage Groups.
+                </Typography>
+              )}
+            </Select>
+          </FormControl>
```

Comment:
Could we add a little margin to the bottom? Looks a little jarring without any. (Note the red)
![image](https://github.com/UoaWDCC/VPS/assets/100455176/f94c2050-ca1f-443f-871a-3b192c48d2f9)

Reply:
![image](https://github.com/UoaWDCC/VPS/assets/161205868/c2daaf40-ab29-49ee-a139-44ca2ce1c8d1)